extras/create_snapshots_reports_scorecard.py

```diff
@@ -924,60 +955,50 @@ def main():
         global reports_to_generate
         # No thread locking is needed here for reports_to_generate because we
         # are still single-threaded at this point
+        logging.info("Building list of reports to generate...")
+        reports_to_generate = create_list_of_reports_to_generate(db, third_party=False)
+
         if args["--no-snapshots"]:
-            # Skip creation of snapshots
+            # Skip creation of "regular" (non-third-party) snapshots
             logging.info("Skipping snapshot creation due to --no-snapshots parameter")
-            reports_to_generate = create_list_of_reports_to_generate(db)
         else:
-            # Generate all necessary snapshots and return the updated list of
-            # reports to be generated
-            reports_to_generate, time_to_generate_snapshots = manage_snapshot_threads(
-                db, cyhy_db_section
+            # Generate all "regular" (non-third-party) snapshots and return the
+            # updated list of reports to be generated
+            reports_to_generate, time_to_generate_snapshots, \
+            time_to_generate_grouping_node_snapshots = manage_snapshot_threads(
+                db, cyhy_db_section, third_party=False
             )
 
         sample_report(
             cyhy_db_section, scan_db_section, nolog
         )  # Create the sample (anonymized) report
         
```

Comment:
This is a bunch of whitespace I noticed. Hoping the suggestions doesn't remove the actual line.

```suggestion

```